en/docs/learn/analytics/securing-sensitive-data-via-the-secure-vault.md

```diff
@@ -0,0 +1,42 @@
+# Encrypting Sensitive Data in the API-M Analytics Server
+
+The WSO2 API Manager comes wth an inbuilt Cipher tool to encrypt sensitive data (e.g., access tokens, passwords, etc.) in  the deployment configuration file. This tool works in conjunction with [WSO2 Secure Vault](https://github.com/wso2/carbon-secvault/blob/master/README.md) to replace sensitive data that is in plain text with an alias. 
+You can use the aliases instead of the actual passwords in your configuration files for better security.The actual value is then encrypted and securely stored in the SecureVault. At runtime, the actual value is retrieved from SecureVault using the alias and then used. 
+
+Follow the steps below to encrypt sensitive data in the deployment configuration file. 
+
+1. Navigate to `<API-M_ANALYTICS_HOME>/conf/worker/secrets.properties`. Enter the required sensitive data element with the value in plain text as a key value pair as shown in the example below. 
+You need to **enter the password after a space to `plainText`** . If not error will be thrown when running the cipher tool.
+
+    ```
+    auth.configs.password=plainText ABC@123
+    ```
+              
+2. Navigate to `<API-M_ANALYTICS_HOME>/conf/worker/deployment.toml`. Enter the alias to be used in the required configuration file instead of the actual value of sensitive data you specified in the previous step. 
+For example if you need to secure the password in Authentication configuration you need to add the configuration as below : 
+    ```
+    # Authentication configuration
+    auth.configs:
+      type: 'local'        # Type of the IdP client used
+      userManager:
+        adminRole: admin   # Admin role which is granted all permissions
+        userStore:         # User store
+          users:
+           -
+             user:
+               username: admin
+               password: ${sec:auth.configs.password}
```

Comment:
as maneesha mentioned in the issue, this value is already a base64 encoded value. Let's add the info on top to avoid confusions.

Reply:
Fixed